cmd/kata-pkgsync/config.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 SUSE LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+obsprojects:
+    home:katacontainers:releases:
+
+        releases:
+            - master
+            - stable-1.8
+            - stable-1.7
+
+        architectures:
+            - x86_64
+
+packagecloud:
+    repo: katacontainers/testing
```

Comment:
the name `testing` is correct ?, or should we choose a better one, like `staging` , `stable`, `beta` .. ?